app/src/main/res/layout-land/continue_interaction_item.xml

```diff
@@ -45,9 +45,9 @@
       android:onClick="@{(v) -> viewModel.handleButtonClicked()}"
       android:text="@string/state_continue_button"
       android:textColor="@color/white"
-      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintBottom_toBottomOf="@+id/previous_state_navigation_button"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintHeight_min="48dp"
-      app:layout_constraintTop_toTopOf="parent" />
+      app:layout_constraintTop_toTopOf="@+id/previous_state_navigation_button" />
```

Comment:
`@id/previous_state_navigation_button`
ditto for others

Reply:
`@+id/previous_state_navigation_button` means creating an id in the namespace. 

`@id/previous_state_navigation_button` means referencing the id which is already defined in the android namespace. 

I don't know why but this is something which Android Studio needs to catch.

Reply:
Okay. I wasn't aware of the difference b/w `id` and `+id` . I'll make the required changes asap. Thank you

Reply:
Thanks @7phalange7. It seems like some of the new IDs aren't using ``@+id`` yet--did you make the requested changes in all of the new locations? Looking at https://github.com/oppia/oppia-android/pull/2197/files shows cases of just ``@id/...``.